recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
+    - maturin
```

Comment:
Add a `# [build_platform != target_platform]` to this line as it should only be used together with `cross-python`.